ext/AvizExt/viz/sensitivity.jl

```diff
@@ -308,7 +310,7 @@ Makie figure
 function ADRIA.viz.outcome_map!(
     g::Union{GridLayout,GridPosition},
     rs::ResultSet,
-    outcomes::NamedDimsArray,
+    outcomes::Dataset,
```

Comment:
Can we make sure this works with a YAXArray as well?

e.g., we just want to plot results for single factor.

Reply:
I'll make this work